common/utils.js

```diff
@@ -355,7 +355,7 @@ const getSaltFromKey = (key) => {
     return keccak256(defaultAbiCoder.encode(['string'], [key.toString()]));
 };
 
-const getContractConfig = async (config, chain) => {
+const getContractStorageConfig = async (config, chain) => {
```

Comment:
I like the previous name better

Reply:
I don't know, I think it's confusing because we usually use the `get_Config` to retrieve something from the json file, but this method is getting the Config from the instantiated contract's storage

Reply:
ah, that's fair, can you add a docstring to clarify this? Maybe we can all it `getAmplifierContractOnchainConfig`, a bit verbose but avoids confusion